client/src/components/VoteButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import HotPastaVoteIcon from '../icons/HotPastaVoteIcon';
+
+const Button = styled.button`
+  border: 1.5px solid grey;
+  border-radius: 15px;
+  box-shadow: 0px 5px 0px #b4b4b4;
+  margin: 10px;
+  outline: none;
+  cursor: pointer;
+  background-color: ${props => (props.disabled ? '#FF9E99' : 'transparent')};
+`;
+
+function VoteButton({ onVoteButtonClick, disabled, pastaPoints }) {
```

Comment:
This `VoteButton` contains a button and a section with "...Pasta Points".
I would rename it